qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -147,12 +147,18 @@ def _default_experiment_options(cls):
             result_index (int): The index of the result from which to update the calibrations.
             group (str): The calibration group to which the parameter belongs. This will default
                 to the value "default".
-
+            save_all_parameter_values (bool): If set to False (the default) then only the
+                calibration parameters for the physical qubits (including default values) and the
+                instructions relevant to the experiment will be saved in the experiment metadata.
+                If this option is set to True then all the values of the parameters in the
+                calibrations will be saved to the metadata, potentially resulting in large
+                metadata volumes.
         """
         options = super()._default_experiment_options()
 
         options.result_index = -1
         options.group = "default"
+        options.save_all_parameter_values = False
```

Comment:
I am tempted to make the default here true. My main hesitation is that ideally we will build a calibration parameter database service that serves this role and then we will want the default here to be false (or just not want this feature any more).

Reply:
I'm also tempted to set this to True but I do not think it would be wise to have this as a permanent solution.

Reply:
With the current implementation the serialization adds all parameter values.